### PR TITLE
Add r-readtext

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by oblute** [comment]
@conda-forge/r @conda-forge/staged-recipes this is ready for review

File: recipes/r-readtext/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('readtext')"           # [not win]
+    - "\"%R%\" -e \"library('readtext')\""  # [win]
+
+about:
+  home: https://github.com/quanteda/readtext
+  license: GPL-3
```
Comment (cbrueffer) [inline recipes/r-readtext/meta.yaml:65]:
```suggestion
  license: GPL-3.0-only
```

File: recipes/r-readtext/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/readtext_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/readtext/readtext_{{ version }}.tar.gz
+  sha256: be0016339eb081a60e26cbe9dd779695378edff8a2ce7b49809bbb94ae950a61
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [r_base == "4.0"]
```
Comment (dbast) [inline recipes/r-readtext/meta.yaml:19]: What is the reason for that skip? Not available dependencies?

Comment (oblute) [reply recipes/r-readtext/meta.yaml:19]: Dependency version inconsistencies/incompatibilities for r-base 4.0.

Comment (cbrueffer) [reply recipes/r-readtext/meta.yaml:19]: The source of that could be the missing `r-antiword` package; possibly gets pulled in from somewhere else for 3.6. Added in https://github.com/conda-forge/staged-recipes/pull/13488 , please try the 4.0 build again once that is merged.

**Comment by cbrueffer** [comment]
The `r-antiword` packages are not available on the CDN yet; I'll restart the CI once they are.

File: recipes/r-readtext/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/readtext_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/readtext/readtext_{{ version }}.tar.gz
+  sha256: be0016339eb081a60e26cbe9dd779695378edff8a2ce7b49809bbb94ae950a61
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (cbrueffer) [inline recipes/r-readtext/meta.yaml:18]: This should be `noarch: generic` now. Given that the Linux build seems to work, this should be mergable afterwards.

Comment (oblute) [reply recipes/r-readtext/meta.yaml:18]: @cbrueffer windows has failed now

Comment (oblute) [reply recipes/r-readtext/meta.yaml:18]: seems like we will need to skip win and r-base 4.0.

Comment (cbrueffer) [reply recipes/r-readtext/meta.yaml:18]: The `noarch: generic` mentioned above takes care of this by building one package that works on all platforms; those packages are built on Linux, the Windows failure doesn't matter then.

Comment (oblute) [reply recipes/r-readtext/meta.yaml:18]: And it allows for windows users to install this in an environment with r 3.6?

Comment (cbrueffer) [reply recipes/r-readtext/meta.yaml:18]: Yes. There'll be one package per R version, should have been more specific.

**Comment by oblute** [comment]
@cbrueffer good to merge?

**Comment by cbrueffer** [comment]
Indeed, thanks!

**Pull request merged**